package.json

```diff
@@ -23,15 +23,15 @@
   "author": "Palantir",
   "license": "Apache-2.0",
   "dependencies": {
-    "stylelint-config-standard": "^16.0.0",
+    "stylelint-config-standard": "^17.0.0",
     "stylelint-order": "^0.3.0"
   },
   "devDependencies": {
-    "stylelint": "7.9.0",
-    "stylelint-scss": "1.4.3"
+    "stylelint": "^8.0.0",
```

Comment:
we don't have a lockfile, so you should pin this version down. you can add yarn in a future PR.